src/setup_payload/AdditionalDataPayloadGenerator.h

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file provides a utility to generate Additional Data payload and its members
+ *      (e.g. rotating device id)
+ *
+ */
+
+#pragma once
+
+#include <core/CHIPError.h>
+#include <system/TLVPacketBufferBackingStore.h>
+
+#define ROTATING_DEVICE_ID_COUNTER_STR_MAX_LENGTH 10
```

Comment:
please use C++-style constants:

```cpp
static constexpr unsigned kRotatingDeviceIdCounterStringMaxLength = 10;
static constexpr unsigned kRotatingDeviceIdHashSuffixLength = 16;
...
```

or even namespace it to avoid smurf naming:

```cpp
namespace RotatingDeviceId {
static constexpr kCounterStringMaxLength = 10;
// ...
```

Reply:
updated